board_rules.md

```diff
@@ -0,0 +1,83 @@
+# Board rules 
+
+The description of the team workflow for the board.
+
+## Description
+
+The board consists of 5 columns:
+
+- `Common backlog`- includes all tasks that should be done.
+- `Sprint backlog` - includes the list of tasks for the current sprint.
+- `In Progress` - the list of tasks that the team works on.
+- `In Review` - the list of tasks that are under review process.
+- `Done` - the list of tasks that successfully finished. If a task contains code, it should be merged to the main branch. In the case, when a task is useless or can not be closed for some resons it should be closed in the next mode "Close as not planned" (won't fix, can't repro, duplicate, stale etc.).
+
+### Common backlog
+
+The process of the issue creating and adding to Common backlog.
+Every issue is placed in a common backlog when it is created.
+The issue should contain a clear title and description with definitions of done (**DoD**).
+Select necessary labels, if applicable. For example, if you add a task to create documentation, put the lable "Documentation" (labels' descriptiopn is provided below). 
+Don't forget to link the project Repository.
+
+### Sprint backlog
+
+Before sprint planning the issues place into Sprint backlog by PM.
+During the sprint planning process, the issues from the sprint backlog are estimated (labels with estimation are assigned).
+
+**Estimation** is carried out with the participation of all team members online using the **Scrum Poker** technology on the resource [planningpokeronline](https://planningpokeronline.com).
+
+Except estimation labels in progect also provided labels: 
+
+- `Invalid`- this doesn't seem right.
+- `Documentation` - improvements or additions to documentation.
+- `Need more discussion`- further information is requested.
+- `Bug`- something isn't working.
+- `Blocker` - urgent level of priority, it's blocking another task.
+- `Blocked` - this task cannot be done until the task on which it depends is completed. (Inside the blocked issue is placed a link to the blocker).
+
+All types of labels are marked with separate color.
+
+**Important**: 
+
+* the fields milestone, and labels with estimation points should be filled for the issues after Sprint planning;
+* if the PR is approved, then the PR creator can independently resolove conversations. 
```

Comment:
Maybe we should move this line to `In review` section.

Reply:
Thanks, fixed it.